src/app/(proper_react)/(redesign)/(public)/how-it-works/page.tsx

```diff
@@ -29,7 +30,13 @@ export default async function Page() {
     typeof oneRepActivations === "undefined" ||
     oneRepActivations > monthlySubscribersQuota;
 
-  if (countryCode !== "us") {
+  const featureFlags = await getEnabledFeatureFlags({
+    ignoreAllowlist: true,
+  });
+
+  console.log("featureFlags", featureFlags);
```

Comment:
```suggestion
```

Reply:
Updated to remove console logs and add flags